test/repeat.jl

```diff
@@ -0,0 +1,39 @@
+module TestRepeat
```

Comment:
Maybe put this into utils.jl instead? The organization of tests is quite messy, but I don't like having one file for each function.